pkg/daemon/ceph/client/crush_rule.go

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
Why not add these to crush.go? It seems simple enough to combine, but that's fine if you prefer this way.

Reply:
As per discussion crush_rule.go s fine.